recipe/meta.yaml

```diff
@@ -25,10 +25,8 @@ requirements:
     - python
     - pickleshare
     - simplegeneric >0.8
-# These are listed in the setup.py, but do not cause the build tests to fail.
-# In order to enable these dependencies, additional tests will need to be defined.
-#    - prompt_toolkit >=0.60
-#    - pygments
+    - prompt_toolkit >=1.0.3,<2.0.0
+    - pygments
```

Comment:
These two are now needed to pass the import test. I am wondering if we should add all the extras listed in `setup.py` too.

```yaml
    # Extras
    - ipyparallel
    - qtconsole
    - ipykernel
    - nbformat
    - notebook
    - ipywidgets
    - nbconvert
    - pathlib2  # [py2k]
    - appnope  # [osx]
    - colorama  # [win]
    - win_unicode_console  # [win] >=0.5
```